server.go

```diff
@@ -475,16 +491,20 @@ func (s *server) handleQuery(querymsg interface{}, state *peerState) {
 		msg.reply <- infos
 
 	case addNodeMsg:
+	case connectNodeMsg:
```

Comment:
I think this breaks the existing `addNodeMsg`, `delNodeMsg` cases because they are no longer being handled. It would probably make sense to keep these cases as they were for compatibility. If not, we could deprecate `addnode` RPC command with a note about using `node` instead.